notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>The code to check endpoints should not require re-running as there is a loop&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Good catch.</p>